src/edu/csus/ecs/pc2/ui/ResultsComparePane.java

```diff
@@ -0,0 +1,461 @@
+
+                selectAndUpdatePrimaryCCSResultsDirectory(primaryCCSResultsDirectoryTextField);
+            }
+        });
+        selectPrimaryCCSResultsDirectoryButton.setToolTipText("Select primary contest results directory");
+
+        JPanel pc2SourcePanel = new JPanel();
+        centerPane.add(pc2SourcePanel);
+
+        exportDirectoryLabel = new JLabel("PC2 Results Directory");
+        exportDirectoryLabel.setToolTipText("");
+        pc2SourcePanel.add(exportDirectoryLabel);
+
+        pc2ResultsDirectoryTextField = new JTextField();
+        pc2ResultsDirectoryTextField.addKeyListener(new KeyAdapter() {
+            @Override
+            public void keyTyped(KeyEvent e) {
+
+                if (e.getKeyCode() == java.awt.event.KeyEvent.VK_ENTER) {
+                    updateUserSetting(ClientSettings.PC2_RESULTS_DIR, pc2ResultsDirectoryTextField.getText());
+                }
+
+            }
+        });
+        pc2ResultsDirectoryTextField.setToolTipText("export pc2 results files directory ");
+        pc2ResultsDirectoryTextField.setColumns(40);
+        pc2SourcePanel.add(pc2ResultsDirectoryTextField);
+
+        JButton selectExportDirectoryButton = new JButton("Select...");
+        pc2SourcePanel.add(selectExportDirectoryButton);
+        selectExportDirectoryButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                selectUpdateExportDirectory(pc2ResultsDirectoryTextField);
+            }
+        });
+        selectExportDirectoryButton.setToolTipText("Select export Directory");
+        
+        JPanel summaryResultsPane = new JPanel();
+        summaryResultsPane.setPreferredSize(new Dimension(800, 300));
+        summaryResultsPane.setMinimumSize(new Dimension(300, 300));
+        summaryResultsPane.setBorder(new TitledBorder(null, "Compare Results Summary", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+        centerPane.add(summaryResultsPane);
+        summaryResultsPane.setLayout(new BorderLayout(0, 0));
+        
+        JPanel resCenterPane = new JPanel();
+        summaryResultsPane.add(resCenterPane, BorderLayout.CENTER);
+        resCenterPane.setLayout(new BorderLayout(0, 0));
+        textArea.setFont(new Font("Courier New", Font.PLAIN, 13));
+        
+        
+//        resCenterPane.add(textArea);
+        
+        JScrollPane scrollPane = new JScrollPane(textArea);
+        resCenterPane.add(scrollPane, BorderLayout.CENTER);
+        
+        JPanel resButtonPane = new JPanel();
+        FlowLayout flowLayout = (FlowLayout) resButtonPane.getLayout();
+        flowLayout.setHgap(45);
+        summaryResultsPane.add(resButtonPane, BorderLayout.SOUTH);
+        
+        JButton viewReportButton = new JButton("View Report");
+        viewReportButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                viewCompareResultsFiles();
```

Comment:
Clicking the `View Report` button _after having previously clicked the `Export Results` button_ generates a report containing an exception (and also generates an exception on the console).  This is definitely a SHOW STOPPER.

Reply:
Updated error handling.  Can you provide steps to reproduce ?

Reply:
I don't know why the steps given in my initial comment aren't sufficient:
```
Clicking the View Report button *after having previously clicked the `Export Results` button* generates a report containing an exception (and also generates an exception on the console). 
```
The steps listed above (and in the original comment) were applied after following the "Steps to Test the PR" given in the initial PR description.  

For completeness, here's a full set of "steps to reproduce the issue" (and yes, the issue still exists):

- Start a clean pc2 server, loading a sample contest using `--load sumithello`.
- Start a pc2 client, login as "admin1".
- Click on the Admin's `Run Contest` tab.
- Click on the `Results Compare` tab.
- To the right of the "Primary CCS Results Directory" textfield, click `Select...`.
- Navigate to, and select, the "domjudge" folder in the `testdata\FileComparisonUtilities\resultscompwork\results` folder.  (In my case the full path to this folder was `C:\Users\john\git\pc2v9-doug-fork\testdata\FileComparisonUtilitiesTest\resultscompwork\results`; it will vary on your machine but should be contained under "testdata" in the pc2 project file hierarchy).
- To the right of the "PC2 Results Directory" textfield, click `Select...`.
- Navigate to, and select, the "pc2" folder in the `testdata\FileComparisonUtilities\resultscompwork\results` folder.  

Note that to this point the above instructions match exactly what is listed in the `Precise steps for testing the PR 
` listed in the initial PR description.

At this point, apply the two steps listed in my comment, _in the specified order_:
1. Click the "Export Results" button.  This generates the following dialog:
![image](https://github.com/pc2ccs/pc2v9/assets/9256691/2d96aa13-3e2a-4934-9479-fdd742220744)
Click `Yes` or `No`; it doesn't matter.
1. Click the "View Report" button.  As stated in my original comment, this produces a report containing an exception:

![image](https://github.com/pc2ccs/pc2v9/assets/9256691/74c495fc-fe43-42c8-b8a5-708d5a795f66)

and it also produces an exception on the console:

```
java.lang.RuntimeException: Invalid results TSV line, too few fields 1
 line: 'Contest not finalized cannot create awards'
 in file C:\Users\john\git\pc2v9-doug-fork\testdata\FileComparisonUtilitiesTest\resultscompwork\results\pc2\results.tsv
   Matching: csus
    at edu.csus.ecs.pc2.core.report.FileComparisonUtilities.validateResultsTSVFileContents(FileComparisonUtilities.java:436)
    at edu.csus.ecs.pc2.core.report.FileComparisonUtilities.createTSVFileComparison(FileComparisonUtilities.java:336)
    at edu.csus.ecs.pc2.core.report.ResultsCompareReport.createReport(ResultsCompareReport.java:250)
    at edu.csus.ecs.pc2.core.report.ResultsCompareReport.writeReport(ResultsCompareReport.java:313)
    at edu.csus.ecs.pc2.core.Utilities.createReport(Utilities.java:659)
    at edu.csus.ecs.pc2.core.Utilities.createReport(Utilities.java:621)
    at edu.csus.ecs.pc2.core.Utilities.viewReport(Utilities.java:725)
    at edu.csus.ecs.pc2.ui.ResultsComparePane.viewCompareResultsFiles(ResultsComparePane.java:411)
    at edu.csus.ecs.pc2.ui.ResultsComparePane$5.actionPerformed(ResultsComparePane.java:160)

```

This is (still) a SHOW STOPPER (or bug in the PR, or whatever you prefer it to be called; it's something that needs to be fixed before I would approve the PR.)